### ISSUE TITLE
Invalid config deadlocks VSCode menus

### ISSUE THREAD

**Issue by RomanKornev** [yatki/vscode-surround#63]
Putting something invalid in `surround.custom`, like:
```jsonc
"surround.custom": {
    "label": "len",
    "description": "len(...)",
    "disabled": false,
    "snippet": "len($TM_SELECTED_TEXT)$0",
}
``` 
and then trying to use Surround hangs the extension. 
As a result, VSCode Command Palette can't function. None of the menus work. Only solution is to kill/restart VSCode itself.

**Comment by yatki** [comment]
Thank you very much for the issue.  I can see that these kinds of problems can be really annoying sometimes.

However, I think putting the right config in the config file is the developer's responsibility. As you might agree, I cannot foresee every invalid config and since the right way is documented in the readme file, unfortunately, I don't think it deserves the effort to fix it. 

Regarding the only solution argument,

> Only solution is to kill/restart VSCode itself.

I think you can also fix the config and move on 🖖 😄 

Have a nice day.

**Comment by yatki** [comment]
@RomanKornev just FYI, I've resolved this issue in version 1.1.1 while adding some new features. Thanks for reporting the issue.

The release is available on: 
- [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=yatki.vscode-surround)
- [GitHub release](https://github.com/yatki/vscode-surround/releases/tag/v1.1.1)

Cheers 🖖,

**Comment by RomanKornev** [comment]
@yatki Thanks, I'll check it out!
